<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QWidget" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="aproch::ANavigationBar" name="widgetNavigationBar" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="MainPageStack">
     <widget class="QWidget" name="BrowserPage"/>
     <widget class="QWidget" name="LibraryPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>aproch::ANavigationBar</class>
   <extends>QWidget</extends>
   <header location="global">Aproch\Include\Widget\ANavigationBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
